nit picks changes to ts files (#577)
huggingface/huggingface_hub: tasks/src/audio-to-audio/about.md

### Speech Enhancement (Noise removal)

> Speech Enhancement is self explanatory. It improves (or enhances) the quality of an audio by removing noise. There are multiple libraries to solve this task, such as Speechbrain, Asteroid and ESPNet. Here is a simple example using Speechbrain

Reviewer: I'd rather keep "a bit" :D it's not that self explanatory. We can keep the language playful in these pages, it doesn't have to be very formal

> Speech Enhancement is self explanatory. It improves (or enhances) the quality of an audio by removing noise. There are multiple libraries to solve this task, such as Speechbrain, Asteroid and ESPNet. Here is a simple example using Speechbrain

Reviewer: ```suggestion
Speech Enhancement is a bit self explanatory. It improves (or enhances) the quality of an audio by removing noise. There are multiple libraries to solve this task, such as Speechbrain, Asteroid and ESPNet. Here is a simple example using Speechbrain
```

### Audio Source Separation

> 

Reviewer: ```suggestion
```

Audio Source Separation allows you to isolate different sounds from individual sources. For example, if you have an audio file with multiple people speaking, you can get an audio file for each of them. You can then use an Automatic Speech Recognition system to extract the text from each of these sources as an initial step for your system!

> Audio-to-Audio can also be used to remove noise from audio files: you get one audio for the person speaking and another audio for the noise. This can also be useful when you have multi-person audio with some noise: yyou can get one audio for each person and then one audio for the noise.

Reviewer: You made a typo with "you"
I'd keep "of multiple people" I don't think there's a term for multi-person
We can keep the Audio-to-Audio
I'd keep "an"s as well.

> Audio-to-Audio can also be used to remove noise from audio files: you get one audio for the person speaking and another audio for the noise. This can also be useful when you have multi-person audio with some noise: yyou can get one audio for each person and then one audio for the noise.

Reviewer: ```suggestion
Audio-to-Audio can also be used to remove noise from audio files: you get one audio for the person speaking and another audio for the noise. This can also be useful when you have audio from multiple people with some noise: you can get one audio for each person and then one audio for the noise.
```

## Training a model for your own data

> 

Reviewer: ```suggestion
```
